include/arch/nrf9160.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system kernel
+ *
+ * Macros and enums for NRF9160 related code
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Damian Loewnau
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef _PHOENIX_ARCH_NRF9160_H_
+#define _PHOENIX_ARCH_NRF9160_H_
+
+
+/* clang-format off */
+enum { gpio_input = 0, gpio_output };
+enum { gpio_low = 0, gpio_high };
+enum { gpio_nopull = 0, gpio_pulldown, gpio_pullup = 3};
```

Comment:
These enums should not be in this header.

Reply:
I see that you like the comment but `enums` are still in this header.

Reply:
Yeah, sorry. Now it is applied.